bootstrap-docker-install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ -f /etc/os-release ]; then
+    # freedesktop.org and systemd
+    . /etc/os-release
+    OS="${ID}-${VERSION_ID}"
+else
+    # We expect RHEL or Centos 7 for now. Lets set something as fallback
+    OS="$(uname -s)-$(uname -r)"
+fi
+
+echo "checking docker executable"
+if ! [ -x "$(command -v docker)" ]; then
+  case $OS in
+  rhel*)
+    extrasrepo=$(cat /etc/yum.repos.d/redhat* | grep -E '^\[.*extras.*\]$' | grep -vE 'debug|source' | tr -d '[|]')
+    sudo yum install -y --enablerepo $extrasrepo docker
+    sudo systemctl daemon-reload
+    sudo systemctl enable docker.service
+    sudo systemctl start docker.service
+    sudo setenforce 0
+    ;;
+  *)
+    echo "installing docker"
+    which yum-config-manager || sudo yum install -y yum-utils
+    sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+    sudo yum install -y docker-ce
+    sudo systemctl daemon-reload
+    sudo systemctl disable firewalld.service
+    sudo systemctl stop firewalld.service
+    sudo systemctl enable docker.service
+    sudo systemctl start docker.service
+    sudo setenforce 0
+    ;;
+  esac
+  echo "sleep 30s to let docker become ready"
```

Comment:
we may should do here, to have a timeout just for the case and not sleep that long if dockerd is not ready yet.
```
timeout -t 30 bash <<EOF
while true; do
  docker info >/dev/null 2>&1
  if [ $? -eq 0 ]; then
    break
  fi
  sleep 10
done
EOF
```